hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -775,6 +776,30 @@ public void deleteKeys(OmDeleteKeys deleteKeys) throws IOException {
 
   }
 
+  /**
+   * Truncates an existing key.
+   *
+   * @param args the args of the key.
+   * @throws IOException
```

Comment:
Would you please finish this java doc, there are many same issue in this PR.

For example `if there is an I/O error while performing the call`